Gemfile

```diff
@@ -5,3 +5,5 @@ source "https://rubygems.org"
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
 gem 'slack-notifier'
+gem 'aws-sdk-kms'
+gem 'base64'
```

Comment:
Lambda のコンソールで教えてくれた。
Client を new するときの Credential と`AWS_LAMBDA_FUNCTION_NAME`は自動で入れてくれるらしい。
```ruby
require 'aws-sdk-kms'
require 'base64'

ENCRYPTED = ENV['sample']
# Decrypt code should run once and variables stored outside of the function
# handler so that these are decrypted once per container
DECRYPTED = Aws::KMS::Client.new
    .decrypt({
        ciphertext_blob: Base64.decode64(ENCRYPTED),
        encryption_context: {'LambdaFunctionName' => ENV['AWS_LAMBDA_FUNCTION_NAME']},
    })
    .plaintext

def lambda_handler(event:, context:)
    # TODO handle the event here
end
```